examples/workflows/plot_simulate_evoked.py

```diff
@@ -109,7 +109,7 @@
 import matplotlib.pyplot as plt
 fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6),
                          constrained_layout=True)
-plot_dipole(dpls, ax=axes[0], layer='agg', show=False)
+plot_dipole(dpls, ax=axes[0], layer='agg')
```

Comment:
I think this should still have `show=False` here since the next line adds another plot to different axes in the same figure.

Reply:
woops ... reverted